lib/hedwig_sms.ex

```diff
@@ -8,7 +8,7 @@ defmodule Hedwig.Adapters.SMS do
   @doc false
   def init({robot, opts}) do
     HTTPoison.start
-    Hedwig.Robot.register(robot, opts[:name])
+    :global.register_name(opts[:name], robot)
```

Comment:
Thanks! I didn't know that about `register_name`. I can give `register_name_external` a try or maybe come up with an alternative